src/standalone/Soil/boundary_conditions.jl

```diff
@@ -155,24 +155,27 @@ presence of a canopy, snow, etc, as in a land surface model,
 this is not the correct method to be using.
 
 This function calls the `surface_fluxes` and `net_radiation`
-functions, which use the soil surface conditions as well as 
+functions, which use the soil surface conditions as well as
 the prescribed atmos and radiation conditions in order to
 compute the surface fluxes using Monin Obukhov Surface Theory.
 """
 function soil_boundary_fluxes(
-    bc::AtmosDrivenFluxBC{<:PrescribedAtmosphere, <:PrescribedRadiativeFluxes},
+    bc::AtmosDrivenFluxBC{
+        <:PrescribedAtmosphere{FT, CA, DT1},
+        <:PrescribedRadiativeFluxes{FT, DT2, T, OD},
+    },
     boundary::ClimaLSM.TopBoundary,
     model::EnergyHydrology{FT},
     Δz,
     Y,
     p,
     t,
-) where {FT}
+) where {FT <: AbstractFloat, CA, DT1, DT2, T, OD}
```

Comment:
Let's remove the extra types since they are not used. same comment re <: AbstractFloat